Running-Mastodon/Heroku-guide.md

```diff
@@ -59,31 +58,32 @@ S3 Region is the AWS code for the region e.g. `ap-northeast-1` not the name of t
 
 To protect the privacy of the users of the your instance, you should have permissons on the your S3 bucket set to no-read and no-write for the public and non-application-specific AWS users, with only one authorized IAM user or group set up to be able to upload or display content. This is an example of an IAM policy used for the S3 bucket used Mastadon instance hentai.loan:
```

Comment:
Mastodon*. This line should be split, too. Maybe we don't want to cite 'hentai.loan' in official documentation either?